bundle/src/main/java/com/adobe/acs/commons/workflow/synthetic/package-info.java

```diff
@@ -21,5 +21,5 @@
 /**
  * Synthetic AEM Workflow APIs.
  */
-@aQute.bnd.annotation.Version("2.0.0")
+@aQute.bnd.annotation.Version("3.0.0")
```

Comment:
is it possible to avoid this major package bump? I think `SyntheticWorkflowRunner` might be missing `@ProviderType`